### DISCUSSION TITLE
Hit rate is 0% for MSVC

### DISCUSSION THREAD

**Discussion by guchangx** [ccache/ccache#793] [Support]
Windows10, VS2019 with ninja use ccache.exe latest.
use default setting, and rebuild cache hit rate is 0%.
![image](https://user-images.githubusercontent.com/73569810/106280399-d1c6db00-6278-11eb-96d9-bebb340a5790.png)
![image](https://user-images.githubusercontent.com/73569810/106280544-09358780-6279-11eb-9fcc-4c268441191e.png)
![image](https://user-images.githubusercontent.com/73569810/106280797-574a8b00-6279-11eb-986e-11c16b068919.png)

Ask an expert to help me see. thanks.

**Comment by jrosdahl** [comment]
That's because ccache doesn't support the MSVC compiler. See [_Supported platforms, compilers and languages_](https://ccache.dev/platform-compiler-language-support.html).
